pom.xml

```diff
@@ -74,6 +74,7 @@
         <groupId>org.jenkins-ci.tools</groupId>
         <artifactId>maven-hpi-plugin</artifactId>
         <configuration>
+          <compatibleSinceVersion>1.15</compatibleSinceVersion>
```

Comment:
We only have this one flag, so it's often applied "creatively" and not strictly based on what it says. In this case, AFAIUI, people may lose permissions, so we really want admins to read the changelog beforehand.